concrete/src/Entity/User/LoginAttempt.php

```diff
@@ -0,0 +1,94 @@
+<?php
+
+namespace Concrete\Core\Entity\User;
+use Doctrine\ORM\Mapping as ORM;
+
+/**
+ * @ORM\Entity(repositoryClass="Concrete\Core\Entity\User\LoginAttemptRepository")
+ * @ORM\Table(
+ *     name="UserLoginAttempts"
+ * )
+ */
+class LoginAttempt
+{
+
+    /**
+     * The attempt ID, this will be a unique identifier
+     *
+     * @ORM\Id @ORM\Column(type="guid")
+     * @ORM\GeneratedValue(strategy="UUID")
```

Comment:
Any reason to use a GUID instead of an autoincrement?

Reply:
Using UUIDs gets us closer to CQRS in the future, is there any reason to use an autoincrement instead of a UUID?

Reply:
Working with integers is faster (less processing overhead) and they require less space/memory...

Reply:
Changing the type of keys once they are in production is not very great...